query_result.go

```diff
@@ -241,6 +242,16 @@ func (qr *QueryResult) parseMap(cell interface{}) map[string]interface{} {
 	return parsed_map
 }
 
+func (qr *QueryResult) parsePoint(cell interface{}) map[string]float64 {
+	point := make(map[string]float64)
+	var array = cell.([]interface{})
+	if len(array) == 2 {
```

Comment:
This condition should always be true, but there is no harm in checking for it.

Reply:
Fair enough, I see parsePath also doesn't check, so I will remove it.